horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/ActionServiceImpl.java

```diff
@@ -127,6 +130,21 @@ public void onNewTest(Test test) {
         executeActions(ActionEvent.TEST_NEW, test.id, test, true);
     }
 
+    @WithRoles(extras = Roles.HORREUM_SYSTEM)
```

Comment:
```suggestion
    @WithRoles(extras = Roles.HORREUM_SYSTEM)
    @Transactional
```

Reply:
- Added `@Transactional`

- No idea by what you mean with "Commit the uncommited changes"